src/Compiler/Service/TransparentCompiler.fs

```diff
@@ -267,6 +267,9 @@ type internal CompilerCaches(sizeFactor: int) =
 
     member val ParseFile = AsyncMemoize(keepStrongly = 50 * sf, keepWeakly = 20 * sf, name = "ParseFile")
 
+    member val ParseFileWithoutProject =
+        AsyncMemoize<string, string, FSharpParseFileResults>(keepStrongly = 10 * sf, keepWeakly = 5 * sf, name = "ParseFileWithoutProject")
```

Comment:
We'd be fine with even a smaller cache in Rider. Maybe even keeping the last file only would already be enough 😅
Maybe make it 5 or something less?

Reply:
Yes, I'm assuming this would really only be used for the single file you are editing right?

Reply:
Yes.

Reply:
Gotcha, made it smaller.